app/views/spree/components/cart/_cart_items.html.erb

```diff
@@ -0,0 +1,21 @@
+<% order = order_form.object %>
+
+<section class="cart-items">
+  <%= render(
+    partial: 'spree/components/cart/cart_item',
+    collection: order.line_items,
+    as: :line_item,
+    locals: { order_form: order_form }
+  ) %>
+
+  <% if (
+    order.adjustments.nonzero.exists? ||
+    order.line_item_adjustments.nonzero.exists? ||
+    order.shipment_adjustments.nonzero.exists? ||
+    order.shipments.any?
```

Comment:
I see this logic eligible for a partial's variable, like `is_order_empty`, WDYT?

Reply:
Great idea, much cleaner 👍